src/main/java/com/crud/tasks/domain/Trello.java

```diff
@@ -0,0 +1,19 @@
+package com.crud.tasks.domain;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Trello {
+    @JsonProperty("board")
+    private int board;
```

Comment:
Warto zastanowić się, czy te pola mogą mieć wartość `null`. Tego oczywiście nie da się wymyśleć - trzeba to sprawdzić w dokumentacji Trello, a jeśli nie ma tej informacji - przyjąć jakieś zalożenia.
Oczywiście jeśli pole może przyjmować wartość `null` , powinno być typu `Integer`, a nie `int`.
Pamiętaj też, że `null` pojawić się może w dwóch sytuacjach:
- jawnie użyty `null` w JSON-ie,
- brak pola w JSON-ie (wtedy też to się mapuje na null!)